### DISCUSSION TITLE
Feature: Format decimal places

### DISCUSSION THREAD

**Discussion by kazad** [kazad/instacalc#56] [Announcements]
Here's another number format, to set the number of output decimal places for a row:

`pi -> 2 decimal places = 3.14`
`pi -> 2 decimals = 3.14`
`pi to 2 decimals = 3.14`

Note: My goal is keeping the input natural (how would you write it, or say it on the phone?). The implied conversion keyword of `to` helps readability. Something like `round(pi, 2)` is workable but unnatural.

**Comment by joshualyon** [comment]
What about both? Coming from other mathematical parsers, `round(x, 2)` is a very common approach. 

In fact, that's how I arrived here. I tried that expression format, assuming it would work based on all the other features of InstaCalc, and when it didn't work I came to check the docs to see if I was missing some detail. :)

**Comment by kazad** [comment]
Good call -- added :).

https://instacalc.com/56906

Generally speaking, I'd like instacalc to figure out what you meant, and people coming from other languages will expect the typical rounding API. Appreciate the suggestion!
